repo/init.go

```diff
@@ -17,7 +17,7 @@ import (
 	"time"
 )
 
-const RepoVersion = "6"
+const RepoVersion = "7"
```

Comment:
Does this value represent the desired migration version? Is there any reason we couldn't use the filenames of the migrations to determine this?

Reply:
This is just what the file is set to when initializing a new node. I suppose you could use the migration  index+1 for it.

Reply:
I'll make an issue for this to track for later fixing.